ios/VisionCameraOcr.swift

```diff
@@ -105,23 +111,52 @@ public class OCRFrameProcessorPlugin: NSObject, FrameProcessorPluginBase {
         ]
     }
     
-    @objc
-    public static func callback(_ frame: Frame!, withArgs _: [Any]!) -> Any! {
+    private static func getBoundingBox(_ rect: CGRect?) -> [String: CGFloat]? {
+         return rect.map {[
+             "left": $0.minX,
+             "top": $0.maxY,
+             "right": $0.maxX,
+             "bottom": $0.minY
+         ]}
+    }
+
+    private func convertOrientation(_ orientation: UIImage.Orientation) -> UIImage.Orientation {
+        switch orientation {
+        case .left:
+            return .right
+        case .right:
+            return .left
+        case .leftMirrored:
+            return .rightMirrored
+        case .rightMirrored:
+            return .leftMirrored
+        default:
+            return orientation
+        }
+    }
```

Comment:
why are you inverting here?

Reply:
In my testing it seemed MLKit uses the opposite meanings of the orientation reported from frame.orientation

If that isn't replicated for anyone else: maybe a better solution would be to introduce an optional orientation parameter to scanOCR

Reply:
Hmm, I guess this is the way of the vision camera is creating the preview

> If that isn't replicated for anyone else: maybe a better solution would be to introduce an optional orientation parameter to scanOCR

that's a good solution indeed

Reply:
It looks like not all devices have this behavior